cord19-explorer/src/main/java/edu/asu/diging/cord19/explorer/core/service/worker/impl/CoordinateParser.java

```diff
@@ -0,0 +1,79 @@
+package edu.asu.diging.cord19.explorer.core.service.worker.impl;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import edu.asu.diging.cord19.explorer.core.model.CoordType;
+import edu.asu.diging.cord19.explorer.core.model.impl.CleanedCoordinatesImpl;
+
+
+public class CoordinateParser {
```

Comment:
this should have an interface in case we ever want to parse the coordinates differently